### PR TITLE
Test results

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/research_experiment_1/issues/60.

### CODE REVIEW

File: app/views/research/submission_test_run_results/_fail.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.test-run-result.open.fail
+.test-run-result.fail
```
Comment (iHiD) [inline app/views/research/submission_test_run_results/_fail.html.haml]: Why has this changed? And the one in `_pass.html.haml`?

Comment (kntsoriano) [reply app/views/research/submission_test_run_results/_fail.html.haml]: The JS collapses everything by default. Did this use to work in a way that it was open by default?

Comment (iHiD) [reply app/views/research/submission_test_run_results/_fail.html.haml]: Yeah. Failed should always be open by default, and passed should always be collapsed by default. So this works to do that by having the different classes in the different files.

Comment (kntsoriano) [reply app/views/research/submission_test_run_results/_fail.html.haml]: Okay.

**Pull request merged**